website/cron.py

```diff
@@ -23,36 +23,36 @@ def unanswered_notification(self):
         import datetime as DT
 
         try:
-            weekago = DT.date.today() - DT.timedelta(days = 6)
+            weekago = DT.date.today() - DT.timedelta(days=6)
             questions = Question.objects\
-                        .filter(date_created__lte = weekago, is_spam = 0)
+                .filter(date_created__lte=weekago, is_spam=0)
         except Exception as e:
             print("No questions found")
         category_count = FossCategory.objects.count()
         i = 0
         body_cat = {}
         for question in questions:
             try:
-                uque = Answer.objects.filter(question__id = question.id)
+                uque = Answer.objects.filter(question__id=question.id)
             except Exception as e:
                 print("error occured >  > ")
                 print(e)
```

Comment:
there should be a continue in the except block. otherwise uque may not be defined afterwards.

Reply:
Currently its not used any where. Need to rewrite this code. As its existing code. We will fix it later as its a independent code.

Reply:
has been rewritten